profiling/simple-kernel-timer/kp_json_writer.cpp

```diff
@@ -57,20 +57,6 @@ inline void write_json(std::ostream& os, KernelPerformanceInfo const& kp,
 }
 // clang-format off
 
-int find_index(std::vector<KernelPerformanceInfo*>& kernels,
-	const char* kernelName) {
-
-	for(unsigned int i = 0; i < kernels.size(); i++) {
-		KernelPerformanceInfo* nextKernel = kernels[i];
-
-		if(strcmp(nextKernel->getName(), kernelName) == 0) {
-			return i;
-		}
-	}
-
-	return -1;
-}
```

Comment:
Please do this cleanup in a separate pull request or at least a separate commit.

Reply:
I'll do it in a separate PR :wink:

Reply:
See #227.